pythoncz/models/github.py

```diff
@@ -125,23 +136,26 @@ def _get_nodes(node, connection_name):
         connection_nodes_count = len(connection_nodes)
         connection_nodes_total_count = node[connection_name]['totalCount']
 
-        if connection_nodes_count > connection_nodes_total_count:
+        if connection_nodes_count < connection_nodes_total_count:
             warnings.warn((
```

Comment:
This is "runtime warning", not warning about usage of obsoleted fuction or similar. This should be handled by logging, not warnings.